receiver/mysqlreceiver/README.md

```diff
@@ -20,7 +20,9 @@ There are also optional metrics that you must specify in your configuration to c
 
 ## Prerequisites
 
-This receiver supports MySQL version 8.0
+This receiver supports MySQL version 8.0.
+
+MariaDB version 10.11 and later also work but features specific to MySQL 8.0 (e.g. the X protocol) will not work.
```

Comment:
`features specific to MySQL 8.0` is that impacting our receiver at all? I don't think we do anything related to MySQL only features. Please clarify as this is going to spook users.

How about:
```suggestion
This receiver supports MySQL version 8.0 and MariaDB 10.11.
```

Reply:
that's good to point out, thank you, I wasn't aware of the distinction.